### PR TITLE
feat: create profile to unzip assembly to local biserver [BACKLOG-40194]

### PR DESCRIPTION
**Pull request by Pancho7**
@pentaho/millenniumfalcon please review

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 10m 52s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 1 test(s) failed:

<details><summary><b>org.pentaho.mantle.client.dialogs.scheduling.validators.RunOnceEditorValidatorTest.testIsValid</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

  
Tests run: **332**, Failures: **1**, Skipped: **12** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pentaho-scheduler-plugin/job/PR-133/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -24,6 +24,7 @@
     <pentaho-scheduler-plugin.version>10.2.0.0-SNAPSHOT</pentaho-scheduler-plugin.version>
     <com.github.spotbugs.annotations.version>4.2.3</com.github.spotbugs.annotations.version>
     <platform.version>10.2.0.0-SNAPSHOT</platform.version>
+    <maven-antrun-plugin.version>1.7</maven-antrun-plugin.version>
```
Comment (afrjorge) [inline pom.xml:27]: Can we use a more recent version (e.g. 3.1.0)?

**Review by afrjorge — APPROVED** [review]

**Review by mbrasil — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (Pancho7) [reply pom.xml]: Done 👍

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 9m 48s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox
```
---
:ok_hand: All tests passed!
 
Tests run: **332**, Failures: **0**, Skipped: **12** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pentaho-scheduler-plugin/job/PR-133/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**